build.c

```diff
@@ -199,6 +202,46 @@ buildadd(struct node *n)
 	e->flags &= ~FLAG_CYCLE;
 }
 
+static void
+statusfmt(char *fmt)
+{
+	struct timespec ctime;
+
+	for (; *fmt; ++fmt)
```

Comment:
The brace style I prefer is to use braces if the body is composed of multiple lines (even if they are just a single statement), or if any of the other branches of the if-else chain needed braces.